code/modules/cargo/packs/security.dm

```diff
@@ -357,3 +357,26 @@
 	cost = CARGO_CRATE_VALUE * 7
 	contains = list(/obj/item/storage/belt/holster/thermal = 2)
 	crate_name = "thermal pistol crate"
+	
+/datum/supply_pack/security/armory/wt550
+    name = "Recalled Weapon Pack"
+    desc = "Contains a set of old nanotranset brand autorifles recalled due to choking hazard."
+	cost = CARGO_CRATE_VALUE * 7
+	hidden = TRUE
+	contains = list(/obj/item/gun/ballistic/automatic/wt550,/obj/item/ammo_box/magazine/wt550m9,
+                    /obj/item/ammo_box/magazine/wt550m9,
+                    /obj/item/gun/ballistic/automatic/wt550)
+    crate_name = "crate"
+	
+/datum/supply_pack/security/armory/wt550ammo
+    name = "Recalled Ammo Pack
+    desc = "Contains four 20-round magazine for the Recalled WT-550 Auto Rifle. Each magazine is designed to facilitate rapid tactical reloads.\
```

Comment:
I think your indentation here isn't working properly. You need to use tab indentation and not space indentation. I'm on my phone so I can't make a suggestion to fix it.